yarn.lock

```diff
@@ -1368,36 +1090,36 @@
 
 "@bentley/bentleyjs-core@^2.19.3":
   version "2.19.28"
-  resolved "https://registry.yarnpkg.com/@bentley/bentleyjs-core/-/bentleyjs-core-2.19.28.tgz#ff612fc9499167f20233f2e89836b3891916e7c3"
-  integrity sha512-M040a5xdCAlkTuzgJioZKt6Az0+OvvbIdOM2AgSM75Fr21IcbyYX2zLok/12AbkFcsnA0H0Jq1xXZUpl9fu8SQ==
+  resolved "https://pkgs.dev.azure.com/bentleycs/_packaging/Packages/npm/registry/@bentley/bentleyjs-core/-/bentleyjs-core-2.19.28.tgz#ff612fc9499167f20233f2e89836b3891916e7c3"
```

Comment:
As the  repo is meant for public consumption, we shouldnt have pkgs.dev.azure.com repository stuff in there.

Reply:
ill update